pkg/server/commands/listusers/list_users.go

```diff
@@ -80,11 +94,12 @@ func fromListUsersRequest(o listUsersRequest) *internalListUsersRequest {
 			ContextualTuples:     o.GetContextualTuples(),
```

Comment:
Something to note here for `UserFilters` and `ContextualTuples`, they share a pointer to the slice. So this is not really a "deep" copy. You still have members/fields that are shared between the incoming request `o` and the returned reference/pointer.

Reply:
The fact that it's not a deep copy is mentioned in the godoc of line 101. At the time I wrote it, I didn't see a need to deep-clone the userFilters and the contextualTuples because I didn't foresee them having to mutate.